enhancements/installer/openstack-no-fip-install.md

```diff
@@ -0,0 +1,78 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@egarcia"
+  - "@adduarte"
+reviewers:
+  - "@mandre"
+  - "@pierre"
+approvers:
+  - "@mandre"
+  - "@pierre"
+creation-date: 2020-5-26
+last-updated: 2020-5-26
+status: implementable
+---
+
+# OpenStack BYO External Connectivity
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+Numerous customers have requested the ability to run the OpenShift on OpenStack IPI installer without using Floating IP addresses.
+In the installer, we use floating IPs as a way to set up exeternal connectivity to the OpenShift cluster. Depending on how your cluster is 
+configured, the usage of floating IPs may not be possible or convenient in OpenStack. Our goal is to support configuring external 
+connectivity to the installer without the use of floating IPs.
+
+
+### Non-Goals
+
+This enhancement does not intend to provide any guidance or features that go further than allowing customers to customize entrypoint connectivity. Operations like attaching loadbalancers to the entrypoints, or any day 2 operations are beyond the scope of this enhancement. 
+
+## Proposal
+
+In order to implement this feature fully, the following requirements must be met:
+    1. Allow users pass pre-configured ports for the Ingress and API VIPs
+    2. Make Floating IP an optional argument in the install-config
+    3. Make External Network an optional argument in the install-config
+
+### User Stories
+
+#### Provider Networks
+
+As an enterprise OpenStack cluster administrator, I don't have VxLAN or GRE tunnel in my environment and am not planning to add them any time soon. I need a way to install the OpenShift IPI installer on provider networks instead. I may also need to clear IP addresses I use on the provider network through an internal audit.
```

Comment:
" I don't have a VxLan or GRE tunnel in my environment..." seems redundant.  aslo it makes no mention of fips. I think this applies to more than just provider network. It allows to install without fips no matter what network right?